python/PyQt/PyQt5/QtGui.py

```diff
@@ -22,3 +22,48 @@
 __copyright__ = '(C) 2015, Matthias Kuhn'
 
 from PyQt5.QtGui import *
+
+
+def __qcolor_repr__(self: QColor):
+    if not self.isValid():
+        return '<QColor: invalid>'
+    elif self.spec() == QColor.Rgb:
+        return '<QColor: RGB {}, {}, {}, {}>'.format(
```

Comment:
@nyalldawson , shouldn't this be RGBA? I think we need to indicate in all of these string representation where the alpha is instead of having ppl guess the 4th nb is alpha.

Reply:
Fair enough!